main.go

```diff
@@ -29,7 +29,7 @@ import (
 
 var (
 	// Version holds the current Gitea version
-	Version = "1.9.0-dev"
+	Version = ""
```

Comment:
perhaps instead:
```suggestion
	Version = "0.0.0-dev"
```

Reply:
why ?

the other variablen (Tags, MakeVersion, ...) are empty too

Reply:
should we fill them up with dummy strings too ?!?

Reply:
@techknowlogick should we fill them up with dummy strings too ?!?

Reply:
It is here so there would be version string if built without make

Reply:
OK